packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,211 @@
+# Nimble Anchor
+
+## Overview
+
+The Nimble Anchor is a component used to navigate to a web resource, similar to the HTML anchor element.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46)<br>
+[Nimble work item](https://github.com/ni/nimble/issues/324)
+
+---
+
+## Design
+
+Initially we will create two flavors of anchor: a standard anchor, and a button-like anchor. These will be two separate components. For justification, see the Future Considerations section at the end of this document.
+
+### API
+
+[FAST's API documentation](https://github.com/microsoft/fast/blob/e576aa70c22780fffba03097277e2db9a2ec1cd8/packages/web-components/fast-foundation/src/anchor/README.md)
+
+-   _Component Name_: `nimble-anchor`
+-   _Properties/Attributes_: We will have the following properties/attributes in addition to the ones provided by the FAST anchor:
+    -   `inline`: specifies that the anchor will be used inline within a text block (as opposed to a standalone element in the UI). The anchor will always have an underline and will be formatted to align with surrounding text.
```

Comment:
I had originally planned on using an attribute name that included "underline", but Jesse suggested we use "inline" to be more abstract and consistent with the intent of the design. That way, if it encompassed more than just the presence or absence of an underline, the name would still fit. In fact, I believe this attribute should also affect the height of the component. The "inline" version should have a height of `auto` so that it fits well within text, but the "standalone" version should have our standard control height so that it aligns well with other controls. 

I'm fine with making "inline" mode the default, but I would lean toward naming the attribute `standalone` (or something similar). @jattasNI do you still feel that way?

Reply:
>  I believe this attribute should also affect the height of the component. The "inline" version should have a height of auto so that it fits well within text, but the "standalone" version should have our standard control height so that it aligns well with other controls.

I disagree more strongly with that. If it's useful that should be a separate component. My previous comment emphasizes that concern for this case too.

> The visuals for buttons and anchors are very different. Buttons are widgets that have a height and width (always a box) while anchors are runs of text that span and can wrap multiple lines. Seems strange for the same control to have such different layout and sizing behaviors.

i.e. changing one of the visual modes should not make it's layout behavior change so significantly. I think it would be very unexpected for users.

Do you have examples of where we need links to behave like a widget instead of a text run?

Reply:
It seemed like the XD doc was specifying a height of 32, though it used the same height for both "standalone" and "inline" versions. I thought that was an oversight. Thinking more about it, I don't see a good reason for the "standalone" version to be 32px high either. Clients can manage the layout themselves to make anchors align with the rest of their UI.

And if both "inline" and "standalone" anchors are the same height, then it makes a lot more sense for our attribute to be called `underline-hidden`. However, here's another consideration: the Breadcrumb links have hidden underlines. I wonder if that is a good enough reason to have our anchor's default be the same, i.e. `underline-visible` instead of `underline-hidden`. It would help us share some CSS, too. What do you think?

Reply:
I talked to Milan offline. He did not feel too strongly either way, leaving the decision up to Brandon's guidance on use cases. Talking to Brandon, he said clients should generally use the standalone version, except (1) when used next to plain text or (2) when there is another need to draw extra attention to it (i.e. as an alternative to using the prominent appearance). From this guidance, it sounds like the default is to have no underline. I'll make the attribute `underline-visible`.